server/config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
```

Comment:
this needs to be in a config var like our DB and Cloudinary credentials. This one is compromised because it's on GitHub so we need to use a new one.